test/integration/addons_test.go

```diff
@@ -70,7 +71,7 @@ func TestAddons(t *testing.T) {
 			args = append(args, "--addons=ingress", "--addons=ingress-dns")
 		}
 		if !arm64Platform() {
-			args = append(args, "--addons=helm-tiller")
+			args = append(args, "--addons=helm-tiller", "--addons=volcano")
```

Comment:
Will the volcano addon not work on arm64? If not we should prevent the user from being able to enable it on if they're on arm64.

Reply:
Volcano's code supports arm64, but volcano is not currently a multi-architecture image, amd64 and arm64 are different tags.  I think that according to the current design of addon, the mirror image of arm64 cannot be pulled, so the use of arm64 is temporarily disabled.  Volcano is advancing the work of mixed mirroring, when volcano supports mixed mirroring, it will open the use of arm64.

Reply:
I will modify the code to disable the use of arm64, thanks for your reminder.

Reply:
I will modify the code to disable the use of arm64, thanks for your reminder.

Reply:
@spowelljr 
I checked the helm-tiller addon and didn't find the code to disable user enablement, am I missing something?

Reply:
And there is one thing I don't understand, We fixed the sha256 value of addon, how about pulling amd64 and arm64 images with a sha256 value?

Reply:
so in order to do that properly, the addon itself will need to be smart enough to know which image to apply based on the user's architecture, which we don't currently have built in to the addon functionality. i'm sure there's a way to do it in the yaml template.